lang/en.json

```diff
@@ -544,8 +544,9 @@
                 "disabled": "Only Non-Featured Products"
             },
             "categories": "Categories",
-            "did_you_mean": "Did you mean: <strong>{suggestion}</strong>",
-            "no_match": "Your search for \"<strong>{query}</strong>\" did not match any products or information.",
+            "did_you_mean": "Did you mean:",
+            "your_search_for": "Your search for",
```

Comment:
The key thing is that a single translation became two translations (in an attempt to get the HTML out--which is admirable).